libcxx/test/std/utilities/function.objects/func.bind.partial/bind_back.verify.cpp

```diff
@@ -0,0 +1,80 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// <functional>
+
+// template<class F, class... Args>
+//   constexpr unspecified bind_back(F&& f, Args&&... args);
+
+#include <functional>
+
+#include "types.h"
+
+void test() {
+  { // Various failures
+    auto p = std::bind_back(pass, 1);
```

Comment:
If it's fundamental that `pass` is an actual function (and not a function object), I would mention it in a comment and I would define `pass` in this test file so we can see its definition immediately.

Reply:
Done, commits: https://github.com/llvm/llvm-project/pull/81055/commits/66632baa67282869fd0de13dd0144ed8a8df7be1, https://github.com/llvm/llvm-project/pull/81055/commits/5d68323a202f4fee6d4ce5beb132e8759732741c